lib/nearest-neighbor.js

```diff
@@ -182,4 +182,28 @@
     return callback(result, maxSimilarity, unmatchedFields);
   };
 
+  exports.findFiveMostSimilar = function(query, items, fields, callback) {
+    var i, item, maxSimilarity, result, similarity, unmatchedFields, _ref;
+    var array = [];
+    maxSimilarity = 0;
+    result = null;
+    i = 0;
+    while (i < items.length) {
+      item = items[i];
+      _ref = recordSimilarity(item, query, fields), similarity = _ref[0], unmatchedFields = _ref[1];
+      item.similarity = similarity;
+      array.push(item);
+      i++;
+    }
+    array.sort(function(obj1,obj2){
```

Comment:
you don't need a new array variable just change result to an array and check the length.